cmd/logcli/main.go

```diff
@@ -19,20 +19,43 @@ import (
 
 var (
 	app        = kingpin.New("logcli", "A command-line for loki.").Version(version.Print("logcli"))
-	quiet      = app.Flag("quiet", "suppress everything but log lines").Default("false").Short('q').Bool()
+	quiet      = app.Flag("quiet", "suppress query metadata").Default("false").Short('q').Bool()
 	statistics = app.Flag("stats", "show query statistics").Default("false").Bool()
-	outputMode = app.Flag("output", "specify output mode [default, raw, jsonl]").Default("default").Short('o').Enum("default", "raw", "jsonl")
+	outputMode = app.Flag("output", "specify output mode [default, raw, jsonl]. raw suppresses log labels and timestamp.").Default("default").Short('o').Enum("default", "raw", "jsonl")
 	timezone   = app.Flag("timezone", "Specify the timezone to use when formatting output timestamps [Local, UTC]").Default("Local").Short('z').Enum("Local", "UTC")
 
 	queryClient = newQueryClient(app)
 
-	queryCmd   = app.Command("query", "Run a LogQL query.")
+	queryCmd = app.Command("query", `Run a LogQL query.
+
+The "query" command is useful for querying for log lines. The default
+output of this command are log entries (a combination of timestamp,
+labels, and a log line) along with various extra information about
+the performed query and its results. Raw log lines (i.e., without a
+label and timestamp) can be retrieved by passing the "-o raw" flag.
+The extra information about the query (API URL, set of common labels,
+excluded labels) can be suppressed with the --query flag.
```

Comment:
"log line" vs "raw log line" is ambiguous. are they the same thing?

can you structure this information more? that would make this clearer I think
maybe something like:
* log line: contains what exactly? only the output lines from the app?
* log entry: timestamp + labels + log line

query output:
* default: log entries + query metadata + result metadata
* `-o raw` to show log lines rather than log entries
* `--query` (you probably meant `--quiet` here!) to surpress query metadata and result metadata